content/en/docs/instrumentation/js/exporters.md

```diff
@@ -121,13 +123,14 @@ Install the exporter package as a dependency for your application:
 npm install --save @opentelemetry/exporter-prometheus
 ```
 
-Update your opentelemetry configuration to use the exporter and to send data to your prometheus backend:
+Update your opentelemetry configuration to use the exporter and to send data to
+your prometheus backend:
 
 ```javascript
-const { PrometheusExporter } = require('@opentelemetry/exporter-prometheus');
-const { MeterProvider }  = require('@opentelemetry/sdk-metrics-base');
+const { PrometheusExporter } = require("@opentelemetry/exporter-prometheus");
```

Comment:
Since this isn't blocking, I'll merge now and let @svrnm and/or @cartermp follow up on this.

Reply:
@pichlermarc, they have been removed because the APIs have changed and so far nobody had the time to write it down with updated once. Let's have a new issue for that.

Reply:
Yeah, we'll absolutely include examples for metrics export in the docs when they're stable. But for now, having older content that won't actually run is a lot worse than not having them in the first place. We had several people report issues about this in the getting started doc, for example.